Razmik_Kirakosyan/python/Pascal/pascal_tringle.py

```diff
@@ -0,0 +1,19 @@
+def pascal_numbers(n):
+	for i in range(0, n):
+
+		for j in range(0, i + 1):
+			print (counting_factorial(i, j),
+					 " ", end="")
+		print ()
+
+def factorial(num):
+	if num == 1 or num == 0:
+		return 1
+	else:
+		return num * factorial(num-1)
+
+def counting_factorial(i, j):
+	return int(factorial(i)/(factorial(i-j)*factorial(j)))
+
+n = int(input())
+pascal_numbers(n)
```

Comment:
- No validation for inputs
![image](https://user-images.githubusercontent.com/80816144/119725501-135f3f80-be81-11eb-8b7f-f7cc82944d40.png)

- For big number
![image](https://user-images.githubusercontent.com/80816144/119725537-1f4b0180-be81-11eb-881c-2ea87bfc3bb7.png)

- Use pep8 style